helpers/tools.ts

```diff
@@ -110,6 +110,30 @@ For better results, you can specify the region parameter to get results from a s
       },
     ],
   },
+  {
+    display: "Artifact generator",
```

Comment:
As this does not generally generate an artifact (e.g no code) - how about calling it Document generator?